hack/ensure-kind.sh

```diff
@@ -30,7 +30,7 @@ goarch="$(go env GOARCH)"
 goos="$(go env GOOS)"
 
 # Note: When updating the MINIMUM_KIND_VERSION new shas MUST be added in `preBuiltMappings` at `test/infrastructure/kind/mapper.go`
-MINIMUM_KIND_VERSION=v0.20.0
+MINIMUM_KIND_VERSION=v0.21.0
```

Comment:
If there will actually be a new kind version. Please also bump the version in tilt.md and quick-start.md